pkg/service/aws/session.go

```diff
@@ -41,7 +41,32 @@ func NewSession(ctx context.Context, region string, accountName string) (*Sessio
 	conf := config.Get()
 	if conf.Env == "local" {
 		log.Println("running in dev mode, using ", conf.AWSAccessID)
-		awsCreds = credentials.NewStaticCredentials(conf.AWSAccessID, conf.AWSSecretKey, conf.AWSToken)
+
+		if conf.AWSAccessID != "" {
+			// creating creds from config variables
+			awsCreds = credentials.NewStaticCredentials(conf.AWSAccessID, conf.AWSSecretKey, conf.AWSToken)
+
+		} else {
+
+			log.Println("AWS_ACCESS_ID is empty, using 'default' profile from user's home directory")
+			// looking for creds in user's home directory with "default" profile
+			sess, err := session.NewSessionWithOptions(
+				session.Options{
+					Profile: "default",
+				},
+			)
+			if err != nil {
+				log.Println("ERROR: provide AWS creds in config file")
```

Comment:
can we add error saying we were not able to find the config  in system or in config.env, something like
 `ERROR: Unable to find the credential config in system or config.env`